src/vs/workbench/contrib/welcomeGettingStarted/browser/media/gettingStarted.css

```diff
@@ -280,7 +280,6 @@
 .monaco-workbench .part.editor>.content .gettingStartedContainer .gettingStartedSlideCategories .getting-started-category {
 	position: relative;
 	border-radius: 6px;
-	border: 1px solid var(--vscode-welcomePage-tileBorder);
```

Comment:
Is this color still used? If not we should also remove it

Reply:
Yes—used here:

https://github.com/microsoft/vscode/blob/df4d9bf75eacb65b8469774372d42794896239f6/src/vs/workbench/contrib/welcomeGettingStarted/browser/media/gettingStarted.css#L411